index.html

```diff
@@ -55,6 +55,9 @@ <h1 id="title">Micheal Jackson</h1>
         </div>
         <a href="https://en.wikipedia.org/wiki/Michael_Jackson" id="tribute-link" target="_blank">more information</a>
     </div>
+    <!-- <div>
+        Lorem ipsum dolor, sit amet consectetur adipisicing elit. Commodi impedit, nihil veniam modi laudantium, animi quia odio rerum consequatur omnis similique suscipit aliquam ad vel sapiente nesciunt dignissimos. Quo earum eos voluptatem commodi, repellat consequuntur voluptas in accusamus corporis aliquam sit vero assumenda distinctio dolorem exercitationem excepturi cupiditate enim quae!
+    </div> -->
```

Comment:
Hi Azeezat Omolola,
You have been doing a tremendous job so far 👌
To Highlight 🎉
GitHub flow is followed. ✔️
README is professional. ✔️

To Improve 🙏
 Please do not leave unused code as a comment in your codebase. Unused code as a comment makes your code not clean.